src/Security/MemberAuthenticator/LoginHandler.php

```diff
@@ -263,7 +264,13 @@ protected function redirectToChangePassword()
             'good'
         );
         $changedPasswordLink = Security::singleton()->Link('changepassword');
+        $changePasswordUrl = $this->addBackURLParam($changedPasswordLink);
+        $whitelistedUrl = Controller::join_links($changedPasswordLink, 'ChangePasswordForm');
```

Comment:
Should `ChangePasswordForm` here be derived somehow?

Reply:
It seems to be hardcoded above in the line 261